### ISSUE TITLE
Add button labels for accessibility

### ISSUE THREAD

**Issue by PatrykMis** [SimpleMobileTools/General-Discussion#203]
Some buttons are not labeled - e.g. back button in settings, no color labels etc.

**Comment by tibbi** [comment]
if you could give a better list of buttons with missing talkbacks, that would definitely help

**Comment by shortwavesurfer2009** [comment]
Well, i only have a few of the apps, but here goes

Simple SMS:
- magnifying glass on the search bar on the main page
- the "+" FAB to create a new conversation on the main page
- all back arrow buttons (except for the one that appears when a message is long pressed.

Simple Files:
- the magnifying glass at the top left of the search bar
- the "+" FAB at the bottom right
- back icons (except when a file or folder has been long pressed

Simple Gallery:
- the magnifying glass icon at the left of the search bar
- all back arrows (except when sokething is selected with a long press

Simple Notes:
- All back buttons

Simple Dialer:
- The magnifying glass on the left side of the search bar
- back button icons
- the delete button above "3" in the dialpad
